bundles/org.openhab.binding.gce/src/main/java/org/openhab/binding/gce/internal/handler/Ipx800DeviceConnector.java

```diff
@@ -72,12 +75,15 @@ public synchronized void send(String message) {
      */
     private void connect() throws IOException {
         disconnect();
+
         logger.debug("Connecting {}:{}...", hostname, portNumber);
-        client = new Socket(hostname, portNumber);
+        Socket client = new Socket(hostname, portNumber);
```

Comment:
```suggestion
        Socket socket = new Socket(hostname, portNumber);
```

Minor but consistent naming woudl be preferred.

Reply:
Changed